.circleci/config.yml

```diff
@@ -74,6 +74,14 @@ jobs:
           root: *workspace_root
           paths:
             - project/build
+            
+  create-layer-directory:
+    executor: node-executor
+    steps:
+      - *attach_workspace
+      - run:
+          name: create layer directory
+          command: yarn copy-node-modules
```

Comment:
Just a note on the naming of this file, I'm not sure it really describes what the command does, maybe something like create or prepare -lambda-layer? Just thinking that it doesn't actually copy anything.

Reply:
Fair point, will rename this.